### PR TITLE
CIV-3459 Cover Proceed Heritage and order made api

### PR DESCRIPTION
**Pull request by karthick-mohanasundaram-HMCTS**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-3459

### Change description ###
1V1 and 1V2
Judge Make the decision and order made ->Order made
Judge Make the decision and accept Strike out Defence Application -> Proceeds in Heritage

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by karthick-mohanasundaram-HMCTS** [comment]
image

**Comment by karthick-mohanasundaram-HMCTS** [comment]
image

**Comment by karthick-mohanasundaram-HMCTS** [comment]
- [ ] Refactor Judge_Makes_Decision to Make_Decision

**Comment by karthick-mohanasundaram-HMCTS** [comment]
image

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "lint": "eslint e2e",
     "test": "npx codeceptjs run --steps",
     "test:smoke": "MOCHAWESOME_REPORTFILENAME=smoke REPORT_DIR=test-results/smokeTest REPORT_FILE=test-results/smokeTest/results.xml npx codeceptjs run --grep @smoke-tests --reporter mocha-multi --verbose",
-    "test:api": "MOCHAWESOME_REPORTFILENAME=api npx codeceptjs run --grep @api-tests --reporter mocha-multi --verbose",
+    "test:api": "npx codeceptjs run --grep @api-tests --reporter mocha-multi --verbose",
```
Comment (karthick-mohanasundaram-HMCTS) [inline package.json:9]: Revert

- [ ] Revert local change

**Comment by karthick-mohanasundaram-HMCTS** [comment]
image

**Comment by karthick-mohanasundaram-HMCTS** [comment]
image

**Review by Azam-Hmcts — DISMISSED** [review]

**Review by deepthidoppalapudihmcts — DISMISSED** [review]

**Review by automatemm — APPROVED** [review]

**Review by Azam-Hmcts — APPROVED** [review]

**Review by MMNycz — APPROVED** [review]

**Review by smitatalawadekar-hmcts — APPROVED** [review]

**Pull request merged**